src/Resources/skeleton/Admin.tpl.php

```diff
@@ -1,28 +1,28 @@
-<?= "<?php\n" ?>
+<?= "<?php\n"; ?>
```

Comment:
How does this relate to your changes?

Reply:
These changes don't are related to the new rule, of course. They are the result of executing `php-cs-fixer` with the existing configuration. I could try to execute the fixer just filtering for the new rule, but I think the codebase should be kept up to date with the provided configuration.
I await for your concerns.

Reply:
Oh you mean running the fixer right now without changes will give that result?

Reply:
That's right.

Reply:
… I tried locally, and those fixes don't appear for me… I upgraded php-cs-fixer first (I'm using 2.14.2)

Reply:
Anyway I'm ok with the changes

Reply:
I have the same version:
```
$ composer global show friendsofphp/php-cs-fixer -N
Changed current directory to /home/jspagnoletti/.composer
name     : friendsofphp/php-cs-fixer
descrip. : A tool to automatically fix PHP code style
keywords : 
versions : * v2.14.2
type     : application
license  : MIT License (MIT) (OSI approved) https://spdx.org/licenses/MIT.html#licenseText
source   : [git] https://github.com/FriendsOfPHP/PHP-CS-Fixer.git ff401e58261ffc5934a58f795b3f95b355e276cb
dist     : [zip] https://api.github.com/repos/FriendsOfPHP/PHP-CS-Fixer/zipball/ff401e58261ffc5934a58f795b3f95b355e276cb ff401e58261ffc5934a58f795b3f95b355e276cb
path     : /home/jspagnoletti/.composer/vendor/friendsofphp/php-cs-fixer
names    : friendsofphp/php-cs-fixer
```
This is my local result in the latest commit (af8723a2b657e4bff1a902edefa251304e3b558c) from `3.x` branch:
```
$ php-cs-fixer fix . --config=.php_cs.dist -v
Loaded config default from ".php_cs.dist".
Using cache file ".php_cs.cache".
Paths from configuration file have been overridden by paths provided as command arguments.
................................................................F........................F..F..................................................................................................................................................................................................................................................FF.
Legend: ?-unknown, I-invalid file syntax, file ignored, S-Skipped, .-no changes, F-fixed, E-error
   1) tests/Fixtures/Admin/ModelAdmin.php (header_comment)
   2) tests/Fixtures/Controller/BatchAdminController.php (strict_comparison)
   3) tests/Fixtures/Controller/ModelAdminController.php (header_comment)
   4) src/Resources/skeleton/AdminController.tpl.php (semicolon_after_instruction)
   5) src/Resources/skeleton/Admin.tpl.php (semicolon_after_instruction)

Fixed all files in 29.760 seconds, 28.000 MB memory used
```

Reply:
Ah I see, I'm using `make cs-fix`, which in turn does `php-cs-fixer fix --verbose`. Using `.` in your command overrides the config about paths.

Reply:
Oh, my bad! Thank you for pointing me the issue. Changes are updated.

Reply:
No problem :)